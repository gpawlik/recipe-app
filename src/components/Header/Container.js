import styled from 'styled-components';

import { colors } from 'variables';

const Container = styled.div`
  background-color: ${colors.MIDNIGHT_BLUE}
`;

export default Container;
