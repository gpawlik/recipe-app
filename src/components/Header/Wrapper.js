import styled from 'styled-components';

const Wrapper = styled.div`
  margin: 0 auto;
  padding: 0 15px;
  max-width: 800px;
`;

export default Wrapper;
