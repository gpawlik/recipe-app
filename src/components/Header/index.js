import React from 'react';

import Container from './Container';
import Wrapper from './Wrapper';
import Logo from 'components/Logo';

const Header = () => {
  return (
    <Container>
      <Wrapper>
        <Logo>Recipe finder</Logo>
      </Wrapper>
    </Container>
  );
};

export default Header;
