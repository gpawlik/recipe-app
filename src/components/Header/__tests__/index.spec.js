import React from 'react';
import { create } from 'react-test-renderer';

import Header from '../';

const initialProps = {};

const mockComponent = props => {
  const finalProps = {
    ...initialProps,
    ...props
  };

  return (
    <Header {...finalProps} />
  );
};

describe('Header', () => {
  it('should render with the default props', () => {
    const tree = create(mockComponent()).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
