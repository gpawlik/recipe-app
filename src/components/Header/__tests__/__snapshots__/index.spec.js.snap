// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render with the default props 1`] = `
<div
  className="sc-bdVaJa lewSnC"
>
  <div
    className="sc-bwzfXH lmIGyu"
  >
    <h1
      className="sc-htpNat jzhphL"
    >
      Recipe finder
    </h1>
  </div>
</div>
`;
